### PR TITLE
:sparkles: Accelerate the e2e setup

### PR DESCRIPTION
**Pull request by yanmxa**
## Summary

- prow pull vm, installed dependency: 11~13 min
- prepare: ocm, policy, app environment in 3 hubs: 15 ~17 min
- ginkgo install global hub operator/operand: 7 min

about 35 min before running the test cases

### Solution

- install the postgres into `hub1`, install the kafka into `hub2`  -> about 2 min
- One main reason is that it takes too much time to pull the image to start the postgres and kafka. So we can pull the images before installing the components!
- [TODO] move global hub installation into shell(asynchronously)?  -> maybe 5 ~ 7 min ?

## Related issue(s)

Fixes #

### CODE REVIEW

File: test/script/e2e_run.sh [hunk 1]
```diff
@@ -41,13 +41,25 @@ options:
...
+    databaseURI: ${database_uri}
     managerImageREF: ${MULTICLUSTER_GLOBAL_HUB_MANAGER_IMAGE_REF}
     agentImageREF: ${MULTICLUSTER_GLOBAL_HUB_AGENT_IMAGE_REF}
     operatorImageREF: ${MULTICLUSTER_GLOBAL_HUB_OPERATOR_IMAGE_REF}
     managedhubs:
 EOF
 
+docker pull "$MULTICLUSTER_GLOBAL_HUB_OPERATOR_IMAGE_REF" &
+docker pull "$MULTICLUSTER_GLOBAL_HUB_MANAGER_IMAGE_REF" &
+docker pull quay.io/stolostron/origin-oauth-proxy:4.9 &
```
Comment (yanmxa) [inline test/script/e2e_run.sh:53]: TODO: align with code image

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**